.gitignore

```diff
@@ -13,7 +13,7 @@ web/css/
 web/media
 web/assetic
 build/
-vendor/
+/vendor/
```

Comment:
Bower doesn't have a lock version yet. The recommended way now is to commit the files.